paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/ACHText.kt

```diff
@@ -0,0 +1,18 @@
+package com.stripe.android.paymentsheet.paymentdatacollection.ach
+
+import android.content.Context
+import com.stripe.android.paymentsheet.R
+
+/**
+ * Temporary hack to get mandate text to display properly until translations are fixed
+ */
+internal object ACHText {
```

Comment:
could we not just fix the strings ourselves? I think you can make edits in lokalise.

Reply:
See thread: https://stripe.slack.com/archives/CFA2HJ99A/p1652290811951309